tests/unit/src/test/scala/tests/testProvider/ZioTestFinderSuite.scala

```diff
@@ -93,7 +150,145 @@ class ZioTestFinderSuite extends FunSuite {
        |""".stripMargin,
     FullyQualifiedName("NestedSpec"),
     Set(
-      ("NestedSpec nested suite nested test", QuickRange(6, 6, 8, 7))
+      Tuple2(
```

Comment:
Why did you add tuples explicitely, that's not needed, could you revert the changes in this file?